pay-api/src/pay_api/exceptions/__init__.py

```diff
@@ -45,7 +46,8 @@ def error_to_response(error: Error, invalid_params=None):
 
 def construct_type(code):
     """Construct type in response."""
-    return 'https://bcrs.gov.bc.ca/.well_known/schemas/problem#{}'.format(code)
+    # return 'https://bcrs.gov.bc.ca/.well_known/schemas/problem#{}'.format(code)
```

Comment:
Stray comment

Reply:
Kept it there, as we may need to re-consider when we move all error codes (all API) to databse.

Reply:
Got it 👍